package.json

```diff
@@ -16,6 +16,7 @@
     "babel-preset-env": "^1.6.0",
     "eslint": "^4.19.1",
     "eslint-config-google": "^0.9.1",
+    "os": "^0.1.1",
```

Comment:
I don't see any usage of os package in this codebase? do you @Rishanthakumar

Reply:
It's a mistake. I'll have that removed.